### DISCUSSION TITLE
Adding a vercel.json file broke my deployments

### DISCUSSION THREAD

**Discussion by Trekiros** [vercel/community#6224] [Help] [answered]
### Summary

Hi!

So, I tried adding a cron job to my nextjs project. I followed the doc to the letter, meaning I added an api handler with the following content:

```src/pages/api/cron.ts

import { doCron } from '@/server/routers/cron'
import type { NextApiRequest, NextApiResponse } from 'next'
 
export default async function handler(req: NextApiRequest, res: NextApiResponse) {
    if (req.headers.authorization !== `Bearer ${process.env.CRON_SECRET}`) {
        return res.status(401).end('Unauthorized');
    }
    
    try {
        await doCron()
        res.status(200).json({ status: 'ok' })
    } catch (err) {
        console.log(err)
        res.status(400).json({ status: 'ko' })
    }
}
```

Then added a vercel.json file with the following content:

```vercel.json
{
    "crons": [
        {
            "path": "/api/cron",
            "schedule": "0 */4 * * *"
        }
    ]
}
```

However, upon pushing the changes, I saw an error message on github with Vercel's icon, telling me "all checks have failed". There's a link that says "details", but clicking it does nothing.
I tried looking at my vercel deployment history and logs, to see if they had more detail, but the new commit doesn't even appear in the deployment history on vercel.

I tried removing the vercel.json file, and suddenly, the commit was deployed as usual. Then I re-created it, and deployments broke once again.
I passed my vercel.json file through both a JSON validator, and a CRON validator, just in case, but both told me everything was okay.

Any idea what I might have missed here? Clearly my vercel.json file is wrong, but I haven't been able to find why and what the fix would be. Thank you for any help!

![image](https://github.com/vercel/community/assets/13800822/798450ac-d4fd-4c49-9690-73738012fc34)
![image](https://github.com/vercel/community/assets/13800822/21e28b96-a58e-4564-9212-d9f84848a4d7)

### Steps to Reproduce

* Fork this repo: https://github.com/Trekiros/questcheck
* Create a Vercel project linked to this repo
* Upon making a new commit, the github repo should say "all checks failed" and vercel won't show a deployment being attempted

**Comment by Trekiros** [comment]
Hopeful post-week-end bump?

**Comment by sebastianroming** [comment, marked as answer]
Are you deploying this project to the hobby plan? Cron Jobs on the Hobby plan can only be executed once a day, and your cron expression tries to execute more than once, so the `vercel.json` file is considered invalid (ref: https://vercel.com/docs/cron-jobs/usage-and-pricing).

**Comment by Trekiros** [comment]
Ah, that was the reason, thank you!
